src/services/transaction/transaction.service.ts

```diff
@@ -6,11 +6,12 @@ import { Session } from "@companieshouse/node-session-handler";
 import ApiClient from "@companieshouse/api-sdk-node/dist/client";
 import { ApiErrorResponse, ApiResponse } from "@companieshouse/api-sdk-node/dist/services/resource";
 
-import { 
+import {
   DESCRIPTION,
   REFERENCE,
   transactionStatuses
 } from "../../config/index";
+import {HttpResponse} from "@companieshouse/api-sdk-node/dist/http/http-client";
```

Comment:
can't see this being used in the code? If not it can come out

Reply:
done